language-support/ts/codegen/tests/build-and-lint.sh

```diff
@@ -2,7 +2,23 @@
 # Copyright (c) 2020 The DAML Authors. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-set -euo pipefail
+set -eo pipefail # Deliberately not set -u here.
+
+# It's sometimes useful to be able to run this test in a filesystem
+# that one can inspect.
+if [[ -z "$BUILD_AND_LINT_TMP_DIR" ]];
```

Comment:
```suggestion
if [[ -z "${BUILD_AND_LINT_TMP_DIR:-}" ]];
```
I think it’s better to be explicit in the one place where a variable might be undefined than allow undefined variables everywhere.

Reply:
Got it. Thanks, that was the bash fu I was lacking.